grails-webhooks/src/test/groovy/webhooks/WebhookServiceSpec.groovy

```diff
@@ -88,6 +89,10 @@ class WebhookServiceSpec extends Specification implements ServiceUnitTest<Webhoo
                 webhookProviderService
             }
         }
+        service.storageService = Mock(MockStorageService){
+            storageTreeWithContext(_) >> {Mock(KeyStorageTree)}
```

Comment:
maybe you can move this to the `then` clause and mark the number of times this should be called (like 1* storageTreeWithContext(_) ...)